chia/consensus/blockchain.py

```diff
@@ -144,7 +143,7 @@ async def create(
         self.constants = consensus_constants
         self.coin_store = coin_store
         self.block_store = block_store
-        self.constants_json = recurse_jsonify(self.constants)
+        self.constants_bytes = bytes(self.constants)
```

Comment:
@arvidn Answering your comment https://github.com/Chia-Network/chia-blockchain/pull/11763#discussion_r888873940 here since its related to this PR:

Retaining the extra field here avoids consensus params conversions for each call into multiprocessing.

Reply:
Conversion goes on here https://github.com/Chia-Network/chia-blockchain/pull/11758#issuecomment-1145913196